pyuvdata/uvdata/ms.py

```diff
@@ -1107,7 +1142,10 @@ def write_ms(
         # this date conversion as few times as possible. Note that the default for MS
         # is MJD UTC seconds, versus JD UTC days for UVData.
         time_array, time_ind = np.unique(self.time_array, return_inverse=True)
-        time_array = (Time(time_array, format="jd").mjd * 3600.0 * 24.0)[time_ind]
+        # TODO: Verify this should actually be UTC, and not some other scale
+        time_array = (Time(time_array, format="jd", scale="utc").mjd * 3600.0 * 24.0)[
```

Comment:
`Time().mjd.to(u.s).value` or `86400` for consistency.

Reply:
Changed to `86400.0` in 5c36be1a.